src/ddmd/backend/code.h

```diff
@@ -426,7 +426,7 @@ unsigned calcblksize (code *c);
 unsigned calccodsize(code *c);
 unsigned codout(int seg, code *c);
 size_t addtofixlist (symbol *s , targ_size_t soffset , int seg , targ_size_t val , int flags );
-void searchfixlist (symbol *s );
+inline void searchfixlist (symbol *s ) {}
```

Comment:
Kept around as noop, as it would be quite some effort to find all the locations in case we ever want to change the mechanism back to emit fixups right after a symbol gets defined.